src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -114,9 +114,9 @@ internal partial class DataFlowPass : AbstractFlowPass<DataFlowPass.LocalState>
         private BitVector _alreadyReported;
 
         /// <summary>
-        /// Reflects the enclosing method or lambda at the current location (in the bound tree).
+        /// Reflects the enclosing member or lambda at the current location (in the bound tree).
         /// </summary>
-        protected MethodSymbol currentMethodOrLambda { get; private set; }
+        protected Symbol currentMember { get; private set; }
```

Comment:
Is it worth it to just call this CurrentSymbol? It's not necessarily a member (possibly a lambda or local function) and now it may not even be a function. This name just seems confusing.

Also, I believe our style guidelines require protected properties to be capitalized.

Reply:
We are more consistent in this area using lowercase spelling (e.g. `topLevelMethod`, `nextVariableSlot`). If we're going to make a change, I'd rather we not do it piecemeal.

---
In reply to: [199903748](https://github.com/dotnet/roslyn/pull/28252#discussion_r199903748) [](ancestors = 199903748)

Reply:
Sounds good.